docs/source/examples/vision/guides/formatters_validating.ipynb

```diff
@@ -0,0 +1,407 @@
+{
```

Comment:
<p>Title is confusing as we use validation also for "ML validation"</p><p>(and the fact that we invented the term "formatters" doesn't help make it clearer)</p><p> </p><p>--</p><p>and the phrasing here isn't good, would shorten it a lot and make it clearer.</p><p> </p><p>Something like:</p><p>"Formatters" are used to transform the structure of your data to the structure required for deepchecks.</p><p>To help ensure your formatters work as they're meant to, deepchecks has built-in helper functions for formatter validation.</p><p> </p><p>This guide will demonstrate the usage of these helper functions on the coco dataset.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/875/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>